main.go

```diff
@@ -226,6 +229,31 @@ func addEnvVars(ciProvider string) {
 	}
 }
 
+// addlFields adds an arbitrary set of fields provided by the end user
+func addlFields() {
+	locn := os.Getenv("BUILDEVENTS_FILE")
+	if locn == "" {
+		return
+	}
+
+	data, err := ioutil.ReadFile(locn)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "unable to read %q: %v\n", locn, err)
+		return
+	}
+
+	err = logfmt.Unmarshal(
+		data,
+		logfmt.HandlerFunc(func(key, val []byte) error {
+			libhoney.AddField(string(key), string(val))
+			return nil
```

Comment:
should just be `return`

Reply:
Hrm? The func signature specifies an error return value, no?

Reply:
derp, I was looking at the return signature for `addfields()` not the handler function.  Nothing to see here...